### PR TITLE
Merge week3-bus-real-time-data to develop

### PR DESCRIPTION
**Pull request by limitimil**
### Functionality
- data fetch for bus metadata `data-fetch/bus.ts`
- render google map and markers
- make data interact with components under week3/ pages

### Stories
- more bus data ( live in data stories now )
```
const BUS_DATA_TYPE = [
  'RealTimeByFrequency',
  'RealTimeNearStop',
  'EstimatedTimeOfArrival',
  'DisplayStopOfRoute',
]
```
- new vuexStory `BusStopMap`

### CODE REVIEW

File: src/data-fetch/busSchedule.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { AxiosResponse } from 'axios'
```
Comment (limitimil) [inline src/data-fetch/busSchedule.ts]: unused file

File: src/services/gMap.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import BusStop from '@/components/_week3Utils/interface/BusStop'
+import { BusDataType, getBusData } from '@/data-fetch/bus'
+
+// TODO: this handler should be refactored as an action of some store
+// eslint-disable-next-line import/prefer-default-export
+export async function drawBusStopOnMap(city: string, routeName: string, direction: string | number, gMapStore: any) {
+  console.log(routeName)
```
Comment (limitimil) [inline src/services/gMap.ts]: remove console log

File: src/week3_bus_data.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Extra Task
```
Comment (limitimil) [inline src/week3_bus_data.md]: remove NOTE

**Review by limitimil — COMMENTED** [review]
## Change Log
Solve issues blow, ready for reviewing

**Review by ygl0118 — APPROVED** [review]

**Pull request merged**